quipucords/api/credential/tests_credential.py

```diff
@@ -146,7 +145,7 @@ def test_hc_create_err_p_or_ssh(self):
         or an ssh_keyfile.
         """
         expected_error = {
-            "non_field_errors": [
+            "non_field_errors": [  # pylint disable:implicit-str-concat
```

Comment:
instead of disabling this check you might fix it. concatenating both strings will do the trick.. The same is valid for the next one.

Reply:
@bruno-fs and @gtanzillo I realized that instead of checking the whole aswer, I could check only if the key "non_field_errors" can be found on response.data. That's why I moved this change from the lint commit to the refactor one.